capstone/src/main/java/com/google/sps/servlets/UsersServlet.java

```diff
@@ -40,96 +40,70 @@
 import javax.servlet.http.HttpServletResponse;
 import javax.sql.DataSource;
 
-@WebServlet("/users")
-public class UserServlet extends HttpServlet {
+@WebServlet("/users/*")
+public class UsersServlet extends HttpServlet {
+  private final String CREATE_NEW_USER_QUERY =
+    "INSERT INTO user_info (user_name, full_name, email, verified, university_id) VALUES (?, ?, ?, true, ?)";
+
+  private final String URL_TO_REDIRECT_AFTER_USER_LOGS_OUT = "/";
+  private final String URL_TO_REDIRECT_AFTER_USER_LOGS_IN =
+    "/users?query_source=server";
+
   private final String GET_USER_ROW_QUERY =
```

Comment:
I do not think you need to keep passing GET_USER_ROW_QUERY as a variable in each method since it is a class variable so all methods should be able to access it without you passing it in the the declaration